--- a/Source/Pcap_DNSProxy/Platform.h
+++ b/Source/Pcap_DNSProxy/Platform.h
@@ -372,7 +372,7 @@
	#define SODIUM_STATIC
 #endif
 #if defined(ENABLE_LIBSODIUM)
-	#include "..\\Dependency\\LibSodium\\sodium.h"
+	#include "LibSodium\\sodium.h"
 #endif
 
 //WinPcap header, always enabled(Windows)
@@ -386,7 +386,7 @@
 	#define HAVE_REMOTE                //WinPcap preprocessor definitions
 #endif
 #if defined(ENABLE_PCAP)
-	#include "..\\Dependency\\WinPcap\\pcap.h"
+	#include "WinPcap\\pcap.h"
 #endif
 
 //Windows API headers

